encord/project.py

```diff
@@ -200,7 +201,16 @@ def add_users(self, user_emails: List[str], user_role: ProjectUserRole) -> List[
         """
         return self._client.add_users(user_emails, user_role)
 
-    def copy_project(self, copy_datasets=False, copy_collaborators=False, copy_models=False) -> str:
+    def copy_project(
+        self,
+        copy_datasets: Union[bool, CopyDatasetOptions] = False,
+        copy_collaborators=False,
+        copy_models=False,
+        *,
+        copy_labels: Optional[CopyLabelsOptions] = None,
```

Comment:
A short line for each arg would be nice to be complete

Reply:
yeah I forgot 🙈

Reply:
ping : )